lib/globals.css

```diff
@@ -263,6 +263,11 @@ svg.tea #steamR {
   display: none !important;
 }
 
+.no-focus:focus {
```

Comment:
What is this? The name is pretty weird.

Reply:
to not have that ugly focus on chakra buttons (it used to happen on single refresh)

Reply:
Maybe tailwind has something. If not, this will do. Pls leave a comment "Hiding focus states from chakra buttons".

Reply:
tried to use "focus:shadow-none focus:outline-none" but chakra overrides tailwind classes, will leave comment